test/controllers/api/rhsm/candlepin_proxies_controller_test.rb

```diff
@@ -402,7 +402,7 @@ def test_regenerate_indentity_certificates
 
         assert_equal @controller.get_parent_host(host_and_capsule), "#{capsule}"
         assert_equal @controller.get_parent_host(just_capsule), "#{capsule}"
-        assert_equal SETTINGS[:fqdn], @controller.get_parent_host(nil_host)
+        assert_equal URI.parse(Setting[:foreman_url]).host, @controller.get_parent_host(nil_host)
```

Comment:
I'm never a fan of having the same code in tests as in the actual code. IMHO it'd be better to have setup code that ensures `Setting[:foreman_url]` is `https://foreman.example.com` and a static assertion:
```suggestion
        assert_equal 'foreman.example.com', @controller.get_parent_host(nil_host)
```

Reply:
@ekohl updated the test to reflect what you wanted to see which makes more sense how you explained it. Does this look ok?